scripts/rabbitmq-defaults

```diff
@@ -41,4 +41,20 @@ ENABLED_PLUGINS_FILE=${SYS_PREFIX}/etc/rabbitmq/enabled_plugins
 
 PLUGINS_DIR="${RABBITMQ_HOME}/plugins"
 
+# Search for another `plugins` directory towards the filesystem
+# root. This is useful when we want to have some common place to put
+# pre-packaged plugins. And $RABBITMQ_HOME usually contains RabbitMQ
+# version somewhere in the middle, making plugin packaging very
+# inconvinient.
+plugins_test_path="$(dirname $(dirname "$PLUGINS_DIR"))"
+while test "$plugins_test_path" != "/"; do
```

Comment:
No, plugins from other versions will be ignored, because we are moving towards fs root and `find` is called with `-maxdepth 1`. - there is now way we can reach sibling directories that way. Using predefined path is simplier, but it'll fail for other standard fs layouts, e.g.:

    /usr/local/lib/rabbitmq/lib/rabbitmq-3.6.6/plugins/
    /usr/local/lib/rabbitmq/plugins/